src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -46,54 +46,29 @@
 
 public abstract class EasyPostResource {
     public static final String EASYPOST_SUPPORT_EMAIL = "support@easypost.com";
-
-    private static final int DEFAULT_CONNECT_TIMEOUT_MILLISECONDS = 30000;
-    private static final int DEFAULT_READ_TIMEOUT_MILLISECONDS = 60000;
-    private static final double APP_ENGINE_DEFAULT_TIMEOUT_SECONDS = 20.0;
-
     public static final Gson GSON =
             new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
                     .registerTypeAdapter(Event.class, new EventDeserializer())
                     .registerTypeAdapter(Rate.class, new RateDeserializer())
                     .registerTypeAdapter(SmartrateCollection.class, new SmartrateCollectionDeserializer()).create();
-
     public static final Gson PRETTY_PRINT_GSON = new GsonBuilder().setPrettyPrinting().serializeNulls()
             .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
             .registerTypeAdapter(Event.class, new EventDeserializer()).create();
-
-    /**
-     * Returns a string representation of the object.
-     */
-    @Override
-    public String toString() {
-
-        return (String) this.getIdString();
-    }
-
-    /**
```

Comment:
# Reviewer Note

This block of code has been moved lower in the file

Reply:
Yeah, this PR still needs a rebase and most of these moving around bits will go away.